frontend/src/components/ModelBox/ModelConfigurationSlider.css

```diff
@@ -1,23 +1,31 @@
 .model-config-slider-fieldset {
 	display: flex;
 	flex-direction: column;
+	margin-top: 0.4rem;
+	padding-top: 0;
+	border: 0.25rem solid var(--control-config-border);
+	border-radius: 0.5rem;
+}
+
+.model-selection-fieldset {
+	min-height: fit-content;
 	margin: 0;
-	padding: 0;
-	border: none;
+	border: 0.25rem solid var(--control-config-border);
+	border-radius: 0.5rem;
 }
 
 .model-config-slider-fieldset legend {
 	margin-right: auto;
 }
 
 .model-config-slider-fieldset .info-icon {
-	margin-top: -1.3rem;
-	margin-bottom: 1rem;
-	margin-left: auto;
+	margin-top: 0.5rem;
 	font-size: 1.3rem;
 }
```

Comment:
Can remove these two (in fact the entire rule block) as we no longer have an info icon here, and it's giving the wrong styles to the details element.

Then you'll need to add this in its place:

```
.model-config-slider-fieldset details > summary {
  list-style: none;
}
```